### ISSUE TITLE
About the Title of the Visualization Window of Pybullet

### ISSUE THREAD

**Issue by ZisongXu** [bulletphysics/bullet3#4470]
Can I change the title name of the Visualization Window of Pybullet? If I can, how? Thanks a lot!

**Comment by Aireos** [comment]
I have the same problem and can't seem to find a way to change the title of the pybullet pop-up to anything other than "Bullet Physics ExampleBrowser using Open GL3 + [btgl] Release build".

**Comment by tovarog23** [comment]
I dont think that can be done specifically in PyBullet unless you modify the API source code and rebuild from there, this is something that can probably be worked around in Bullet SDK instead.
